TalkAloud/TalkAloud/Cells/AudioRecordingCell.swift

```diff
@@ -22,11 +22,10 @@ class AudioRecordingCell: UITableViewCell {
             for tag in tags {
                 allTags += tag + ", "
             }
-            allTags.removeFirst()
             
             allTags = allTags.trimmingCharacters(in: .whitespaces)
             
-            if tags.count > 1 {
+            if tags.count >= 1 {
```

Comment:
This approach was more common in the ObjC days.
On Swift you have isEmpty
https://developer.apple.com/documentation/swift/array/1688398-isempty

Reply:
I do come from a C++ background so I guess this is why 😅